ops/testing.py

```diff
@@ -173,7 +173,7 @@ def add_relation(self, relation_name, remote_app, remote_app_data=None):
         #       remote_app_data isn't empty.
         return rel_id
 
-    def add_relation_unit(self, relation_id, remote_unit_name, remote_unit_data=None):
+    def add_relation_unit(self, relation_id, remote_unit_name, *, remote_unit_data=None):
```

Comment:
We need to discuss the idea of these extra parameters, per discussion on that other PR, sooner rather than later, otherwise people will start trusting the parameters and we will drive people mad by breaking their tests.